templates/site/test-button.html

```diff
@@ -0,0 +1,46 @@
+{% extends "layouts/main.html" %}
+{% set page_title = "Test Buttons" %}
+
+{% block content %}
+    
+    {% with med_action_button = true%}
+        {% set button_disabled = "" %}
```

Comment:
Make this the default so it doesn't have to be explicitly stated every time for non-disabled buttons.

Reply:
Disabled status made default.